CHANGELOG.md

```diff
@@ -4,13 +4,14 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 # Upcoming release
 
 ### Added
-- **UITableViewExtentions**:
-  - Added `isValidIndexPath(_ indexPath:)` method to check whether given IndexPath is valid within UITableView. [#441](https://github.com/SwifterSwift/SwifterSwift/pull/441) by [setoelkahfi](https://github.com/setoelkahfi).
-**Optional**:
+- **UITableViewExtensions**:
```

Comment:
One final thing before we merge this, @setoelkahfi could you please update this with **UITableView**

Reply:
done @omaralbeik